bundles/org.eclipse.cdt.lsp.clangd/src/org/eclipse/cdt/lsp/internal/clangd/editor/ClangdConfigurationArea.java

```diff
@@ -169,4 +183,16 @@ void dispose() {
 		texts.clear();
 	}
 
+	public boolean optionsChanged(ClangdOptions options) {
```

Comment:
package visibility looks sufficient

Reply:
Sorry, I was wrong. It's called in the `ClangdConfigurationPage`, so it must be public.